dev/devicelab/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   args: ^0.13.4
   meta: ^1.0.4
   path: ^1.4.0
-  process: 2.0.1
+  process: 2.0.3
```

Comment:
Yes. process depends on platform. When I bumped platform I also had to update process. Otherwise pub's dependency solver is unhappy.